### PR TITLE
update datasets version for docker

### PR DESCRIPTION
**Pull request by VincyZhang**
## Type of Change

update datasets version for docker build

### CODE REVIEW

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/NeuralChat/job/ITREX-Chatbot-Inference/88/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/view/NeuralChat/job/ITREX-Chatbot-Finetune-HPU/98/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/ITREX-Chatbot-Inference-HPU/76/

File: examples/huggingface/pytorch/language-modeling/nas/README.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ Neural architecture search (NAS) training script [`run_mlm.py`](./run_mlm.py) is
 Recommend python 3.9 or higher version.
 ```shell
 pip install -r requirements.txt
-pip install transformers==4.34.1
+pip install
```
Comment (chensuyue) [inline examples/huggingface/pytorch/language-modeling/nas/README.md:12]: typo?

**Pull request merged**